NSURL+IDN.podspec

```diff
@@ -4,7 +4,7 @@ Pod::Spec.new do |s|
   s.license   = 'MIT'
   s.summary   = 'Support for IDN (punycode) in NSURL'
   s.homepage  = 'https://github.com/wordpress-mobile/NSURL-IDN.git'
-  s.authors    = { 'Jorge Bernal' => 'jbernal@gmail.com', 'Sérgio Estêvão' => 'sergio.estevao@automattic.com'}
+  s.authors    = { 'WordPress-Mobile' => 'mobile@automattic.com'}
```

Comment:
Actually, can we use a space instead of `-`?

Reply:
Or `WordPress Mobile Team`

Reply:
How about just WordPress? Thinking of every other platform they don't specify whether they're mobile or not. Facebook Mobile or Pinterest Mobile isn't a thing. We're just the mobile team within the WordPress org.